src/main/scala/com/cognite/sdk/scala/v1/resources/timeSeries.scala

```diff
@@ -97,4 +85,16 @@ object TimeSeriesResource {
     deriveEncoder[TimeSeriesSearch]
   implicit val timeSeriesQueryEncoder: Encoder[TimeSeriesQuery] =
     deriveEncoder[TimeSeriesQuery]
+  implicit val assetsFilterEncoder: Encoder[TimeSeriesFilter] =
```

Comment:
Isn't it strange that we didn't need this before?

Reply:
Looks like it's because `Filter.filterWithCursor` requires `Encoder[TimeSeriesFilter]` while the previous approach just added the `assetIds` to the url.